plugins/obs-ffmpeg/obs-ffmpeg-nvenc.c

```diff
@@ -356,11 +356,53 @@ static void nvenc_destroy(void *data)
 	bfree(enc);
 }
 
+#include "external/nvEncodeAPI.h"
+
+typedef NVENCSTATUS(NVENCAPI *NV_MAX_VER_FUNC)(uint32_t *);
+extern void *load_nv_func(const char *func);
+extern bool nv_failed(obs_encoder_t *encoder, NVENCSTATUS err, const char *func, const char *call);
+
+bool check_driver_version(obs_encoder_t *encoder)
```

Comment:
Should we also use this function inside the `jim_nvenc` encoder to not duplicate the code?

Reply:
this code was a part of init_nvenc_internal of a new nvenc. 

it did not changed how i was expected. so it makes sense to just extract it to a new function and use in both encoders.